mod/missions/views/default/missions/element-select.php

```diff
@@ -116,6 +116,22 @@
             ));
         	break;
             
+        case elgg_echo('missions:opt_in'):
+        	$content .= elgg_view('input/dropdown', array(
+                'name' => $dropdown_name . '_element',
+                'value' => '',
+                'options' => array(
```

Comment:
These are the opt-in options available on GCcollab, there are more available for GCconnex that are missing.